### DISCUSSION TITLE
Will there be any problems with the rolling upgrade from version 2.4 to version 2.9?

### DISCUSSION THREAD

**Discussion by hexiang1219** [grafana/mimir#5434] [Help and support]
#### I want to do a mimir upgrade from 2.4 to 2.9
current environment：
    PVC used by store-gateway and ingester components. Using an external object storage service

Will there be any problems with rolling upgrades?

**Comment by pstibrany** [comment]
Please see the [release notes](https://github.com/grafana/mimir/releases) for individual releases and changelog for list of breaking changes (`[CHANGE]` items) between releases.
